data/names.txt

```diff
@@ -3518,6 +3448,165 @@ phrase "pirate"
 		"Fate"
 		"Boys"
 
+phrase "pirate"
+	phrase
+		"historical male pirates"
+		"historical female pirates"
+		"historical pirate ships"
+		"historical ghost ships"
+		"fictional male pirates"
+		"fictional female pirates"
+		"fictional pirate ships"
+		"fictional ghost ships"
+
+phrase "historical male pirates"
+	word
+		"Blackbeard"
+		"Edward Teach"
+		"William Kidd"
+		"Long Ben"
+		"Henry Morgan"
+		"Francois l'Olonnais"
+		"Francis Drake"
+		"Cheung Po Tsai"
+		"Bartholomew Roberts"
+		"Benjamin Hornigold"
+		"Calico Jack"
+		"Emanuel Wynn"
+		"Christopher Moody"
+		"Stede Bonnet"
+		"Paulsgrave Williams"
+		"Dragut"
+		"Thomas Tew"
+		"Henry Every"
+		"Philip Lyne"
+		"Kanhoji Angre"
+
+phrase "historical female pirates"
+	word
+		"Lady Killigrew"
+		"Ching Shih"
+		"Anne Bonny"
+		"Mary Read"
+		"Johanna Hård"
+		"Charlotte Badger"
+		"Catherine Hagerty"
+		"Sadie the Goat"
+		"Rachel Wall"
+		"Maria Lindsey"
+		"Sayyida al Hurra"
+		"Lo Hon-cho"
+		"Jeanne de Clisson"
+		"Grace O'Malley"
+		"Teuta of Illyria"
+		"Jeanne de Clisson"
+		"Christina Anna Skytte"
+		"Anne Dieu-le-Veut"
+
+phrase "historical pirate ships"
+	word
+		"Queen Anne's Revenge"
+		"Whydah"
+		"Wyndham Galley"
+		"Revenge"
+		"Adventure Galley"
+		"Quedagh Merchant"
+		"Adventure Prize"
+		"Golden Fleece"
+		"Black Prince"
+		"Rising Sun"
+		"The William"
+		"La Nouvelle Trompeuse"
+		"Sea Nymph"
+		"Bachelor's Delight"
+		"Rose Pink"
+		"Scowerer"
+		"Sultana"
+		"Ranger"
+		"Lark"
+		"Postillion"
+		"Good Fortune"
+		"Royal Fortune"
+		"Marianne"
+		"Amity"
+		"Pembroke"
+		"Margaret"
```

Comment:
Some of these single word pirate ships were on the old list of pirate names that I moved down to this list. I would be okay with removing some of the less memorable ship names if we like.